### ISSUE TITLE
ui-textbox - Feedback should not be shown if no-errors

### ISSUE THREAD

**Issue by murbanowicz** [JosephusPaye/Keen-UI#128]
Hi,

I found that:

```
<div class="ui-textbox-feedback">
            <div class="ui-textbox-error-text ui-textbox-feedback-toggle-transition" style="display: none;"></div>
            <div class="ui-textbox-help-text ui-textbox-feedback-toggle-transition"></div>
            <!--v-if-->
        </div>
```

It is giving additional padding which is unnecessary and looks bad.
V-if in should be placed in parent div, not in child ones only.

**Comment by JosephusPaye** [comment]
This is similar to #124.

**Comment by JosephusPaye** [comment]
Fixed in v1.0. If there is no `error` or `help`, the feedback `div` is not shown.
